<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbulEduWidgetLOMV1</class>
 <widget class="QWidget" name="AbulEduWidgetLOMV1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
 border: 1px solid gray;
 border-radius: 5px;
 margin-top: 2ex;
}
 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center;
     padding: 0 3px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbLomProperties">
     <property name="title">
      <string>Indexation LOM (1/2)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QScrollArea" name="saLomProperties">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>889</width>
           <height>499</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frGeneral">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_11">
             <property name="margin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>4</number>
             </property>
             <item row="0" column="0">
              <layout class="QGridLayout" name="glLOM1_5">
               <property name="spacing">
                <number>4</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="lblLOM1_5">
                 <property name="text">
                  <string>1.5 Mot clé libre</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="cbLOM1_5"/>
               </item>
               <item row="0" column="2">
                <widget class="QLineEdit" name="leLOM1_5"/>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="btnPlusLOM1_5">
                 <property name="maximumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>+</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <layout class="QGridLayout" name="glLOM1_10">
               <property name="spacing">
                <number>4</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="lblLOM1_10">
                 <property name="text">
                  <string>1.10 Typologie générale de document</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="cbLOM1_10"/>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="btnPlusLOM1_10">
                 <property name="maximumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>+</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frTechnique">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="margin">
              <number>4</number>
             </property>
             <property name="spacing">
              <number>4</number>
             </property>
             <item row="0" column="0">
              <layout class="QGridLayout" name="glLOM4_1">
               <property name="spacing">
                <number>4</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="lblLOM4_1">
                 <property name="text">
                  <string>4.1 Format</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="leLOM4_1"/>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="btnPlusLOM4_1">
                 <property name="maximumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>+</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <layout class="QGridLayout" name="gridLayout_3">
               <property name="spacing">
                <number>4</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="lblLOM4_2">
                 <property name="text">
                  <string>4.2 Taille du fichier</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="lblKo">
                 <property name="text">
                  <string>Ko</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="lblLOM4_7">
                 <property name="text">
                  <string>4.7 Durée</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <spacer name="hsLOM4_7">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="1">
                <widget class="QTimeEdit" name="teLOM4_7"/>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="leLOM4_2"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frAuthenticate">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblAuthenticate">
        <property name="text">
         <string>Vous devez être authentifié pour modifier ou compléter ces informations</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="hsAutheticate">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>260</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnAuthenticate">
        <property name="text">
         <string>S'authentifier</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>saLomProperties</tabstop>
  <tabstop>cbLOM1_5</tabstop>
  <tabstop>leLOM1_5</tabstop>
  <tabstop>btnPlusLOM1_5</tabstop>
  <tabstop>cbLOM1_10</tabstop>
  <tabstop>btnPlusLOM1_10</tabstop>
  <tabstop>leLOM4_1</tabstop>
  <tabstop>btnPlusLOM4_1</tabstop>
  <tabstop>leLOM4_2</tabstop>
  <tabstop>teLOM4_7</tabstop>
  <tabstop>btnAuthenticate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
